src/create_disk.sh

```diff
@@ -333,8 +333,8 @@ install_uefi() {
     mkdir -p "${vendordir}"
 	cat > ${vendordir}/grub.cfg << 'EOF'
 search --label boot --set prefix
-set prefix=($prefix)
-configfile $prefix/grub2/grub.cfg
+set prefix=($prefix)/grub2
+configfile $prefix/grub.cfg
```

Comment:
One very important thing we need to keep in mind here is even after bootupd, this config file is "unmanaged state"; we don't have a tool that owns updating it.

So we need to be careful making any changes that depend on it being updated.

(Arguably bootupd should own this too, if we reach consensus on that happy to have that filed as an issue)

Reply:
Yup.  The `create_disk.sh` change is a cleanup with no expected behavioral change, so that's okay, and the `grub.cfg` change should be strictly additive.  Once bootupd starts reinstalling BIOS GRUB and FCCT switches `/boot` to 1.2 metadata, the specified `$boot` and `$prefix` values will be the default, so setting them in old `grub.cfg`s should be harmless.